soc/nxp/imx/Kconfig.soc

```diff
@@ -0,0 +1,10 @@
+# Copyright 2024 NXP
+# SPDX-License-Identifier: Apache-2.0
+
+config SOC_FAMILY_IMX
+	bool
+
+config SOC_FAMILY
+	default "nxp_imx" if SOC_FAMILY_IMX
```

Comment:
```suggestion
config SOC_FAMILY_NXP_IMX
	bool

config SOC_FAMILY
	default "nxp_imx" if SOC_FAMILY_NXP_IMX
```

Reply:
I suggest to use SOC_FAMILY_IMX because there are some existing device drivers are using this configuration item. so need to modify soc.yml and :

config SOC_FAMILY
	default "imx" if SOC_FAMILY_IMX

Reply:
Or update the drivers as has been done for other vendors, should be a simple `sed` command

Reply:
done - s/SOC_FAMILY_IMX/SOC_FAMILY_NXP_IMX